src/Symfony/Component/Console/Helper/Table.php

```diff
@@ -237,13 +244,27 @@ public function setHeaders(array $headers)
         return $this;
     }
 
+    /**
+     * Sets the rows.
+     *
+     * @param array $rows
+     *
+     * @return Table
+     */
     public function setRows(array $rows)
     {
         $this->rows = array();
 
         return $this->addRows($rows);
     }
 
+    /**
+     * Adds a list of rows.
+     *
+     * @param array $rows
```

Comment:
Added, I used the same message used in addRow function 
`A row must be an array or a TableSeparator instance.`